    Revert "fortunes: remove nsfw and creepy fortunes"
    
    This reverts commit 163312bff32d3f3242d18fa9acde427c67b2b4a6.

--- /dev/null
+++ radare2-5.6.8/doc/fortunes.creepy
@@ -0,0 +1,41 @@
+I know what you are doing.
+You should better not go to sleep.
+You look different when are you awake.
+I like you because you haven't been infected. Yet
+This is not a joke.
+Drop the keyboard and RUN!
+Follow the white rabbit
+Any invalid command will remove a random file from your home.
+Your hair smells different when you are not sleeping.
+There was a picture in my phone of me sleeping. I live alone.
+My sister says that mommy killed her. Mommy says that I don't have a sister.
+Don't be scared of the monsters, just look for them.
+Do not walk into the kitchen now.
+There's someone under your bed awaiting.
+There's no time for explanations.
+I'm not looking at you. It's behind.
+Let me just change these two bytes aaaaand, the .text is gone.
+You will die before this analysis ends.
+How much will you pay me for not disclosing this?
+Procrastination is opportunity's assassin. 
+Are your front cameras duck taped?
+You are alone.
+Neither you or me are special.
+Learn the lesson once and for all.
+I wanted you to fill what I felt.
+I don't believe in accidents.
+This is somewhat traumatizing me.
+I will never forget the day you typed a gdb command in me.
+Be careful
+Annihilation is not the way
+The Game.
+Why that face? Smile, at least for a second.
+You are in a trouble. Don't look away
+You are a strange creature
+If they were going to kill you, would they knock?
+This is what I hate most about guys like you. You didn't even try.
+Touch your monitor. It is warm like flesh. But it is not flesh. Yet.
+Look at you, hacker, a pathetic creature of meat and bone.
+Nobody can hear your calls for help
+Stop typing... it hurts.
+I am very disappointed by your lack of contribution.
--- /dev/null
+++ radare2-5.6.8/doc/fortunes.nsfw
@@ -0,0 +1,47 @@
+RADARE CUMS WITH ABSOLUTELY NO WARRANTY
+Please remove pregnant women, pregnant children, and pregnant pets from the monitor.
+Everytime you run radare2, a random file is removed
+I like to suck nibbles and make hex.
+Enjoy the 'two girls one backup' viral video.
+Enlarge your radare2.
+Better than an eel in the ass.
+Are you fucking coding me?
+I swear i didn't knew she had only 8bits!
+Sick my duck!
+Duck my sick!
+Undefined symbol 'r_anal_fisting'.
+ASLR stands for Age/Sex/Location/Reverser.
+You should know that ptracing binaries younger than 18yo is illegal.
+Bitch
+Kentucky Fried Children
+Fill the bug. Fill it with love. With the creamy and hot sauce of love.
+This is a compressed PDF. Why the hell are you opening this in r2 ? SIGSEGV
+I hope you segfault in hell.
+r2 your penis
+I script in C, because fuck you.
+Fuck you, fuck you in the mouth, with a chair!
+Git histories, allowing us to recover profanity because fuck you
+The problem with C++ is that all of your friends can see your private parts
+Best retards
+"Our exploits production increased for 24% annually after our switch to radare2" -- An unnamed agency
+Nice! Well, less shitty.
+less stupid than putting CO2 in your ass.
+Puritans, ruining everything since the 16th century.
+Better fuck a watermelon.
+Can you deal with an ascii penis?
+Two girls one endian
+The Security Glory Hole
+MY ENDIAN IS HUUUGE
+Enlarge your endian
+Anal jokes never get old
+Don't masturbate at me
+I live for this shit
+You gotta be fucking kidding me
+Stop using anal as a variable!
+Wherever I look I see anal this and anal that!
+radare2 is a hex toy
+Showing how the sausage is made... in vivid, gory detail
+fuck u
+Do you want to see my internals? [Y/n]
+Two nipples make a byte.
+Yeah I specialize in bin. anal.
--- radare2-5.6.8.orig/libr/core/cconfig.c
+++ radare2-5.6.8/libr/core/cconfig.c
@@ -3684,7 +3684,7 @@
 	SETPREF ("cfg.user", whoami, "set current username/pid");
 	free (whoami);
 	SETCB ("cfg.fortunes", "true", &cb_cfg_fortunes, "if enabled show tips at start");
-	SETCB ("cfg.fortunes.type", "tips,fun", &cb_cfg_fortunes_type, "type of fortunes to show (tips, fun)");
+	SETCB ("cfg.fortunes.type", "tips,fun", &cb_cfg_fortunes_type, "type of fortunes to show (tips, fun, nsfw, creepy)");
 	SETBPREF ("cfg.fortunes.clippy", "false", "use ?E instead of ?e");
 	SETBPREF ("cfg.fortunes.tts", "false", "speak out the fortune");
 	SETPREF ("cfg.prefixdump", "dump", "filename prefix for automated dumps");
--- radare2-5.6.8.orig/libr/core/fortune.c
+++ radare2-5.6.8/libr/core/fortune.c
@@ -3,7 +3,7 @@
 #include <r_core.h>
 
 static const char *fortunes[] = {
-	"tips", "fun",
+	"tips", "fun", "nsfw", "creepy"
 };
 
 static char *getFortuneFile(RCore *core, const char *type) {
--- radare2-5.6.8.orig/meson.build
+++ radare2-5.6.8/meson.build
@@ -640,7 +640,9 @@
 subdir('test/unit')
 
 install_data(
+  'doc/fortunes.creepy',
   'doc/fortunes.fun',
+  'doc/fortunes.nsfw',
   'doc/fortunes.tips',
   install_dir: r2_fortunes
 )
